src/components/Share/PetCardItem.tsx

```diff
@@ -0,0 +1,58 @@
+import Image from "next/image";
+import { z } from "zod";
+import Button from "./Button";
+import { IconPetGender } from "@/icons/IconPetGender";
+import { cn } from "@/utils/helpers";
+
+const petCardItemSchema = z.object({
+  name: z.string(),
+  gender: z.string(),
+  image: z.string().url(),
+});
+
+interface PetCardItemProps extends z.infer<typeof petCardItemSchema> {}
+
+export const PetCardItem: React.FC<PetCardItemProps> = ({
+  name,
+  gender,
+  image,
+}) => {
+  return (
+    <div
+      className="flex max-w-[242px] flex-col items-center rounded-[32px] border px-[16px] py-[24px] leading-[1.5] shadow-card-shadow tablet:max-w-[275px] tablet:py-[32px] desktop:px-[32px] desktop:py-[24px]"
+      aria-label={`${name} - ${gender === "female" ? "samiczka" : "samiec"}`}
+    >
```

Comment:
Consider using `<figure>` and  `<figcaption>` tags for better accessibility.

Reply:
fixed